nilearn/_utils/docs.py

```diff
@@ -73,7 +73,22 @@ def custom_function(vertices):
     If `None`, the complete figure is used.
 """
 
-# bg_img
+# surf_mesh
+docdict["surf_mesh"] = """
+surf_mesh : :obj:`str` or :obj:`list` of two :class:`numpy.ndarray` \
```

Comment:
FYI I think we try to keep the entries in this file in alphabetical order within each section, so this should probably be moved down